msteams-platform/tabs/how-to/authentication/auth-tab-aad.md

```diff
@@ -45,7 +45,7 @@ Authentication flow should be triggered by a user action. You shouldn't open the
 
 Add a button to your configuration or content page to enable the user to sign in when needed. This can be done in the tab [configuration](~/tabs/how-to/create-tab-pages/configuration-page.md) page or any [content](~/tabs/how-to/create-tab-pages/content-page.md) page.
 
-Azure AD, like most identity providers, doesn't allow its content to be placed in an `iframe`. This means that you'll need to add a pop-up page to host the identity provider. In the following example, this page is `/tab-auth/simple-start`. Use the `authentication.authenticate()` function of the Microsoft Teams client SDK to launch this page when the button is selected.
+Azure AD, like most identity providers, doesn't allow its content to be placed in an `iframe`. This means that you'll need to add a pop-up page to host the identity provider. In the following example, this page is `/tab-auth/simple-start`. Use the `authentication.authenticate()` function of the Microsoft Teams client library to launch this page when the button is selected.
```

Comment:
Please check if we need to update line 10 in this md file.to "> For authentication to work for your tab on mobile clients, ensure that you're using version 1.4.1 or later of the TeamsJS client library."

Reply:
Thank you! Fixed.